editor/utils/mobile.js

```diff
@@ -0,0 +1,14 @@
+
+const IS_MOBILE = window && window.innerWidth < 782;
```

Comment:
I don't know that there's a great solution here, but the maintainability of this being kept in sync with `variable.scss`'s `$break-medium` is concerning.

Reply:
I noticed that we set the following class on PHP side:
```php
if ( wp_is_mobile() ) {
	$admin_body_class .= ' mobile';
}
```
I was wondering if that could be useful in this context.